charts/kubeflow-trainer/values.yaml

```
@@ -0,0 +1,108 @@
+#
+# Copyright 2025 The Kubeflow authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Default values for trainer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- String to partially override release name.
+nameOverride: ""
+
+# -- String to fully override release name.
+fullnameOverride: ""
+
+jobset:
+  # -- Whether to install jobset as a dependency managed by trainer.
+  # This must be set to `false` if jobset controller/webhook has already been installed into the cluster.
+  install: true
+
+# -- Common labels to add to the resources.
+commonLabels: {}
+
+image:
+  # -- Image registry.
+  registry: ghcr.io
+  # -- Image repository.
+  repository: kubeflow/trainer/trainer-controller-manager
+  # -- Image tag.
+  # @default -- If not set, the chart version will be used.
+  tag: ""
```

Otherwise, the default installation fails.

```suggestion
  # -- Image tag.
  tag: latest
```

---

Historically, we try to keep `latest` tag in the `master` branch and release tag in the `release-xx` branch.
Like in Katib:
- master branch: https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/installs/katib-standalone/kustomization.yaml#L23
- release branch: https://github.com/kubeflow/katib/blob/release-0.18/manifests/v1beta1/installs/katib-standalone/kustomization.yaml#L23

Shall we do the same for Helm Charts  @ChenYi015 @Electronic-Waste @astefanutti @kubeflow/wg-training-leads?